app/.env.test

```diff
@@ -44,3 +43,4 @@ REACT_APP_PAGINATION_SIZE=1000
 REACT_APP_USER_FORM_HIDDEN_FIELDS=""
 REACT_APP_USER_FORM_RENDER_FIELDS=""
 REACT_APP_DEFAULTS_TABLE_PAGE_SIZE=10
+REACT_APP_SENTRY_CONFIG_JSON="{"DSN": "https://something@sentry.onalabs.org/12345"}"
```

Comment:
use whole context also

Reply:
Will remove this since its not being used